css/index.css

```diff
@@ -1,9 +1,182 @@
-/* Example of @import use
-This imports the `reset.css` file */
-@import "./reset.css";
+@import "https://fonts.googleapis.com/css?family=Poppins:200i,400&display=swap";
+
+:root {
+  --color-white: #f3f3f3;
+  --color-darkblue: #1b1b32;
+  --color-darkblue-alpha: rgb(27 27 50 / 80%);
+  --color-green: #37af65;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
 
-/* For testing purposes, delete this code */
 body {
```

Comment:
You can use the shorthand property for font here

Reply:
ok boss, thanks for the review. pls, i did understand the shorthand property you are talking about